cargo/spec/dependabot/cargo/file_parser_spec.rb

```diff
@@ -521,8 +521,8 @@
         end
       end
 
-      context "that is unparseable" do
-        let(:manifest_fixture_name) { "unparseable" }
+      context "that is unparsable" do
+        let(:manifest_fixture_name) { "unparsable" }
```

Comment:
This is an actual file, so if we make the switch we also need to rename the file. According to google, both seem correct and unparseable more commonly used, so maybe we can keep it as is and add an ignore for it.

Reply:
ok, ignored, force-pushed rerun of codespell